wordpress-mobile/WordPress-Android
# Post Settings - Tags

Fixes #5808. This PR adds a new tags screen. Similar to excerpt and slug PRs, it doesn't handle the design changes inside the settings page. There might be some improvements to make in `EditPostSettingsFragment` but I am specifically staying away from making too many changes there for now. Once we have the other components, I am planning on a full review of that fragment to make the necessary improvements all together.

To test:
* Go into tags screen in post settings and make sure it's filtered correctly as you type.
* Selecting a tag should add it at the end.
* Duplicate tags are not handled for now. 
* Going back should update the tags text view in settings and updating the post should update the remote tags.

Just a reminder that, there is a v2 issue for tags #5921 and if there are any improvement ideas outside of the scope of this PR, we should consider adding them into that issue.

_Known issue:_ There is a jump when you go back from the tags page, I'll fix this when I work on the settings fragment.

![screenshot_1495652580](https://cloud.githubusercontent.com/assets/662023/26420971/020db8dc-40cd-11e7-9475-4b4ac5c77b09.png)
